### PR TITLE
Update to latest cli-core

### PR DESCRIPTION
**Pull request by dprothero**
Updates the README to be accurate and updates all dependencies to their latest versions.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,9 @@ Access and stream your Twilio debugger logs along with your calls and messages.
 # Usage
 <!-- usage -->
```
Comment (childish-sambino) [inline README.md]: Drop the `usage` tags or oclif will just re-create this section. Example: https://github.com/twilio/plugin-debugger/commit/c7340c6c2edd19818fade58136be436f9c6afa73#diff-04c6e90faac2675aa89e2176d2eec7d8

Comment (dprothero) [reply README.md]: I removed the command from package.json, but I should do that too.

Comment (childish-sambino) [reply README.md]: Yeah, saw that after. Any reason to remove it?

Comment (dprothero) [reply README.md]: It's incorrect for plugins. Notice what it had for the installation instructions:
`npm install -g @twilio-labs/plugin-watch`

Comment (childish-sambino) [reply README.md]: Yup, but the rest is correct. Just dropping the `usage` tags and using the custom `usage` works.

**Review by micaswyers — APPROVED** [review]
🚢

File: README.md (hunk 1 above)
Comment (dprothero) [reply README.md]: TIL... thanks for the tip

**Review by micaswyers — APPROVED** [review]

**Pull request merged**